src/backend/functions/setup/s3-handler.js

```diff
@@ -1,20 +1,30 @@
 const {
   API_GATEWAY,
+  BUILD_BUCKET,
   COGNITO_IDENTITY_POOL,
   FROM_BUCKET,
   MAX_TASKS,
   REGION,
   TO_BUCKET
 } = process.env;
 
+const BACKEND_PATH = "docker/backend.zip";
 const CONFIG_FILENAME = "settings.js";
 const FROM_PREFIX = "static/";
 
 module.exports = s3 => {
   const copyFile = params => s3.copyObject(params).promise();
-  const deleteFile = params => s3.deleteObject(params).promise();
+  const deleteFiles = params => s3.deleteObjects(params).promise();
```

Comment:
Moved to delete files in order to do a single call to s3. In addition to it, that is the way to remove multi-versioned objects (the code Pipeline source bucket needs to be a versioned bucket).

Reply:
Also here I was wrong, the correct way is to list all the versions, and then delete all of them, so I changed this